arrow-data-source/common/src/main/java/com/intel/oap/spark/sql/execution/datasources/v2/arrow/SparkManagedAllocationListener.java

```diff
@@ -20,24 +20,55 @@
 import org.apache.arrow.memory.AllocationListener;
 
 public class SparkManagedAllocationListener implements AllocationListener {
+    public static long BLOCK_SIZE = 8L * 1024 * 1024; // 8MB per block
```

Comment:
how about make this configurable?

Reply:
That's a good idea.

Although at this time I think a fixed size is enough for most cases: say 8M can already reduce Spark `acquire` calls by 99%, in my own measurement. 

Also the similar logic was applied to C++ Arrow code so it might require further thinking to unify them somehow. On the whole I think we can open a individual topic for the suggestion. :)